package.json

```diff
@@ -55,5 +56,8 @@
   },
   "browserslist": [
     "since 2017-06"
-  ]
+  ],
+  "dependencies": {
+    "aws-sdk": "^2.1643.0"
+  }
```

Comment:
THis is a dev dependency. This will get bundled in our tracker

Reply:
Duly noted, thank you